doc/guides/maintaining-npm.md

```diff
@@ -10,52 +10,20 @@ are at the discretion of the release and LTS teams.
 This process only covers full updates to new versions of npm. Cherry-picked
 changes can be reviewed and landed via the normal consensus seeking process.
 
-## Step 1: Clone npm
+## Step 1: Run the update script
 
 ```console
-$ git clone https://github.com/npm/cli.git npm
-$ cd npm
+$ ./tools/update-npm.sh
 ```
 
-or if you already have npm cloned make sure the repo is up to date
+## Step 2: Commit new npm
 
 ```console
-$ git remote update -p
-$ git reset --hard origin/latest
-```
-
-## Step 2: Build release
-
-```console
-$ git checkout vX.Y.Z
-$ make
-$ make release
-```
-
-Note: please run `npm dist-tag ls npm` and make sure this is the `latest`
-**dist-tag**. `latest` on git is usually released as `next` when it's time to
-downstream
-
-## Step 3: Remove old npm
-
-```console
-$ cd /path/to/node
-$ git remote update -p
-$ git checkout -b npm-x.y.z origin/master
-$ cd deps
-$ rm -rf npm
-```
-
-## Step 4: Extract and commit new npm
-
-```console
-$ tar zxf /path/to/npm/release/npm-x.y.z.tgz
-$ git add -A npm
+$ git add -A deps/npm
 $ git commit -m "deps: upgrade npm to x.y.z"
```

Comment:
Maybe that would be a good idea to add those instructions in the script itself? The script could replace `x.y.z` to the correct version, helping the maintainer even more.

Reply:
Hmmm, thought I responded but I guess it got lost.

I experimented with including all the git stuff inside the script but for some reason checking out the branch didn't work right away. The more I thought about it the more I realized it might not make sense to automate the git stuff since you also need to fetch the remote and rebase with whitespace fix... and figured it might be better to not do any of the git stuff and keep it as a separate task

Reply:
> figured it might be better to not do any of the git stuff and keep it as a separate task

💯 agree with that. What I meant was to have the bash script print those commands to stdout, but let the maintainer type them themself – the way `git node land` does when instructing to push to upstream is a pretty great UX IMHO.

Reply:
OH, that's a cool idea.